libqtile/widget/check_updates.py

```diff
@@ -56,29 +56,37 @@ def __init__(self, **config):
                          "Mandriva": ("urpmq --auto-select", 0)
                          }
 
-        # Check if distro name is valid.
-        try:
-            self.cmd = self.cmd_dict[self.distro][0].split()
-            self.subtr = self.cmd_dict[self.distro][1]
-        except KeyError:
-            distros = sorted(self.cmd_dict.keys())
-            logger.error(self.distro + ' is not a valid distro name. ' +
-                         'Use one of the list: ' + str(distros) + '.')
-            self.cmd = None
+        # Backwards compatibility for custom_command. Convert string to tuple.
+        if not isinstance(self.custom_command, (list, tuple)):
+            self.custom_command = (self.custom_command, 0)
+            logger.warning("string inputs to custom_command are deprecated." +
+                           "please use tuple: (\"command\",int(lines to strip from output))")
+
+        if not self.custom_command[0]:
+            # Check if distro name is valid.
+            try:
+                self.cmd = self.cmd_dict[self.distro][0]
+                self.subtr = self.cmd_dict[self.distro][1]
+            except KeyError:
+                distros = sorted(self.cmd_dict.keys())
+                logger.error(self.distro + ' is not a valid distro name. ' +
+                             'Use one of the list: ' + str(distros) + '.')
+                self.cmd = None
+                self.subtr = 0
+        else:
+            # Otherwise use custom_command
+            self.cmd = self.custom_command[0]
+            self.subtr = self.custom_command[1]
```

Comment:
Technically, you could simplify this to
```py
self.cmd, self.subtr = self.custom_command
```
if you like.

Reply:
I'll add `[:2]` to custom_command in case the user inputs a list with more than two elements.